spec/unit/block_score/api_connection_error_spec.rb

```diff
@@ -1,12 +1,10 @@
-module BlockScore
-  RSpec.describe APIConnectionError do
-    let(:stubbed_route_pattern) { %r{.*api\.blockscore\.com/people/abc123} }
-    let(:stubbed_error) { Errno::ECONNREFUSED }
-    let(:message) { 'Connection refused - Exception from WebMock' }
-    subject { -> { BlockScore::Person.retrieve('abc123') } }
+RSpec.describe BlockScore::APIConnectionError do
+  let(:stubbed_route_pattern) { %r{.*api\.blockscore\.com/people/abc123} }
+  let(:stubbed_error) { Errno::ECONNREFUSED }
+  let(:message) { 'Connection refused - Exception from WebMock' }
+  subject { -> { BlockScore::Person.retrieve('abc123') } }
```

Comment:
The `subject` is evaluated lazily, so there is probably no need to return a `Proc` object here. I realize the shared context would need to be changed but there's only 3 uses that would need to change too.